ASCIT/donut
# Zmo/committee sites

### Summary
- Created the committee_sites. 
- Create a page editing system. One can edit the committee sites as well as create new pages if they have the privilege. 
- A user can upload files to the site if needed.  
- Currently, anyone can edit pages, but it can easily be edited such that only people with admin permission can access those links. 

### Test Plan
- Create a new page in the editor. Check that the page exists, and check that it exists in the 'all page created' page. Check that the file can be deleted. 
- Upload a file and check that it exists in the ‘all uploads’ page. Check that the file can be deleted.
